chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/reviewer/FieldLengthReviewer.java

```diff
@@ -26,27 +30,36 @@ public void review(CertifiedProductSearchDetails listing) {
         if (listing.getVersion() != null && !StringUtils.isEmpty(listing.getVersion().getVersion())) {
             checkField(listing, listing.getVersion().getVersion(), "productVersion");
         }
+        if (listing.getSed() != null && listing.getSed().getTestTasks() != null) {
+            for (TestTask task : listing.getSed().getTestTasks()) {
+                checkField(listing, task.getUniqueId(), "taskIdentifier");
+                for (TestParticipant tp : task.getTestParticipants()) {
+                    checkField(listing, tp.getUniqueId(), "participantIdentifier");
+                }
+            }
+        }
     }
 
     private void checkField(final CertifiedProductSearchDetails product, final Object field, final String errorField) {
         if (field instanceof Long) {
             Long fieldCasted = (Long) field;
             if (fieldCasted.toString().length() > getMaxLength("maxLength." + errorField)) {
                 CertifiedProductSearchDetails productCasted = (CertifiedProductSearchDetails) product;
```

Comment:
I know this isn't new code, but just wondering why the cast to CertifiedProductSearchDetails is being done since the passed-in "product" is already of that type? Can you clean this up a bit?